server/utils/bedspaceUtils.test.ts

```diff
@@ -142,4 +152,308 @@ describe('bedspaceV2Utils', () => {
       })
     })
   })
+
+  describe('summaryList', () => {
```

Comment:
The Summary List-related tests below have been moved from the test suite for the Bedspace Service, where the implementation originally resided.